workspaces/libnpmpublish/lib/get-pathname.js

```diff
@@ -0,0 +1,22 @@
+'use strict'
```

Comment:
Did this really need a separate file?  Given the kinds of bugs we've been finding the last few weeks consolidating these out of the cli I would humbly ask if this could be inline and tested as part of unpublish.js itself.

Reply:
💯